doc/sphinx/source/quickstart/installation.rst

```diff
@@ -171,29 +204,41 @@ Note that the ESMValTool source code is contained in the ``esmvaltool-python``
 package, so this package will always be installed as a dependency if you install
 one or more of the packages for other languages.
 
-There is also a lesson available in the
-`ESMValTool tutorial <https://esmvalgroup.github.io/ESMValTool_Tutorial/>`_
-that describes the installation of the ESMValTool in more detail. It can be found
-`here <https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html>`_.
+.. _install_julia_dependencies:
+
+Installation of Julia dependencies
+----------------------------------
 
+If you want to use the ESMValTool Julia functionality, you will also need to
+install Julia. If you are just getting started, we suggest that you
+come back to this step later when and if you need it.
+To perform the Julia installation, make sure that your conda
+environment is activated and then execute
+
+.. code-block:: bash
+
+    mamba install julia
+    esmvaltool install Julia
 .. _install_on_macosx:
 
-Installation on MacOSX
-======================
+ESMValTool installation on MacOS
+---------------------------------
 
-The Python diagnostics of the ESMValTool are supported on MacOSX, but Julia, NCL,
-and R are not. If any of these are needed, deployment through a :ref:`Docker<install_with_docker>`
+The Python diagnostics of the ESMValTool are supported on MacOS, but Julia,
+NCL, and R are not.
+If any of these are needed, deployment through a
+:ref:`Docker<install_with_docker>`
 container is advised.
 
 The ``esmvaltool-python`` diagnostics can be installed as follows:
 
-First, ensure mamba is pre-installed (see `Mamba/Conda installation`_ for more details).
+First, ensure mamba is pre-installed (see install_with_mamba_ for more details).
 
 Create a new environment with the ``esmvaltool-python`` package:
 
 .. code-block:: bash
 
-    mamba create --name esmvaltool esmvaltool-python 'python=3.10'
+    mamba create --name esmvaltool esmvaltool-python
```

Comment:
OK but unsecured since the user might have downloaded the 3.8 package then they'll complain the Python they use is old; note that during a base update, conda doesn't up the Python version

Reply:
But we're creating a new environment here, which will be populated with the latest versions of everything, not updating the ``base`` environment.

Reply:
yeah but that'll keep the Python that `base` came with, which is in fairness pinned in `environment` so it can't be lower than 3.8, but that will still use older pythons if they've installed eg miniconda-3.8, hence up-to-date packages but that are built with the old Python; this is more of a policy question here - do we want to encourage users to always use the latest Python or we leave them to take that decision themselves? I'm more inclined to force them to use the latest Python we support

Reply:
> yeah but that'll keep the Python that base came with

Are you sure? When I look at the environments I created on my own computer, they all have newer Python versions than the one that is in the `base` environment.

Reply:
I am quite sure about the major version ie major for Python being 3.8 or 3.9 in my books - it will indeed update minor versions ie from 3.9.4 to 3.9.7. Let me test this one more time now and let you know, bud

Reply:
nope - am wrong - it updates from 3.9 to 3.10 no problem, not sure of an older mamba, and miniconda3 with py3.8 but we tell users to always update their base so we good! :+1: